examples/parachain-example/examples/create_nft.rs

```diff
@@ -0,0 +1,141 @@
+//!
+//! # Setting up the local parachain
+//!
+//! We use [zombienet](https://github.com/paritytech/zombienet) to start up a local asset hub.
+//!
+//! ## 1. Install necessary tools
+//!
+//! To setup the local parachain we need to have 3 binaries installed in our path:
+//!
+//! ### 1. `zombienet`
+//!
+//! Zombienet is a tool for quickly spinning up a (local) blockchain network. Please follow the install guide in the [zombienet github repo](https://github.com/paritytech/zombienet).
+//!
+//! ### 2. `polkadot`
+//!
+//! Build the polkadot binary from the [polkadot github repo](https://github.com/paritytech/polkadot) and install it in your path:
+//! ```txt
+//! git clone https://github.com/paritytech/polkadot.git
+//! cd polkadot
+//! cargo build --release
+//! cargo install --path .
+//! ```
+//!
+//! ### 3. `polkadot-parachain`
+//!
+//! The polkadot asset hub is part of the [cumulus github repo](https://github.com/paritytech/cumulus), an SDK for developing parachains.
+//! Building the cumulus workspace produces a binary called `polkadot-parachain` that has the capability and configuration data to run the asset hub.
+//! ```txt
+//! git clone https://github.com/paritytech/cumulus.git
+//! cd cumulus
+//! cargo build --release
+//! cargo install --path .
+//! ```
+//!
+//! ## 2. Run the parachain locally
+//!
+//! Zombienet can now spawn the parachain locally from a configuration file, `asset-hub-zombienet.toml` in this case.
+//! We need to have at least 2 validator nodes running (via the `polkadot` binary),
+//! while the `polkadot-parachain` binary starts the asset hub parachain and gets it registered with the validator nodes.
+//! To do that, run:
+//! ```txt
+//! zombienet -p native spawn asset-hub-zombienet.toml
+//! ```
+//! Zombienet uses Kubernetes by default, but we can use it without Kubernetes, by providing the `-p native` flag.
+//!
+//! You might have noticed, that we use `chain = "rococo-local"` in the `asset-hub-zombienet.toml` file for the relay chain.
+//! This is just to make the epoch time shorter and should have no effect on your interactions with the parachain.
+//! Polkadot / Kusama / Rococo have different epoch times of `24h` / `2h` / `2min` respectively.
+//! The parachain is only registered after the first epoch. So we need to wait 2 minutes, until the parachain becomes interactive and produces blocks.
+//!
+//! # Running the example
+//!
+//! After you have the network up and running, you should see something like [https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:42069#/explorer]
+//! as the polkadot.js link to the parachain (e.g. "collator01") in the zombienet output.
+//!
+//! In this example we use the _uniques_ pallet of the polkadot asset hub parachain.
+//! The dev account _alice_ creates an NFT collection, then mints an NFT that is part of the collection.
+//! Beware that this example can take up to a minute to run. Specify the port like so:
```

Comment:
nit: should a configurable `--port 10101` be provided in the example?

Reply:
Ah, I forgot to remove this line. The port is fixed in the example (hard coded in the `asset-hub-zombienet.toml` and the `create_nft.rs` files).